packages/material-ui/src/ListItem/ListItem.js

```diff
@@ -12,6 +12,12 @@ import useEnhancedEffect from '../utils/useEnhancedEffect';
 import useForkRef from '../utils/useForkRef';
 import ListContext from '../List/ListContext';
 import listItemClasses, { getListItemUtilityClass } from './listItemClasses';
+import { listItemButtonClasses } from '../ListItemButton';
+import ListItemSecondaryAction from '../ListItemSecondaryAction';
+
+// TODO remove these in v6
+// - button related (styling and logic) like autoFocus, focusVisible, disabled
+// - container
```

Comment:
These TODO items are now scattered throughout the code. Let's create an issue instead to remove noise from the code.